R/utilities-ratio-comparison.R

```diff
@@ -107,29 +113,36 @@ getPKParameterRatioPKTable <- function(data,
   for (set in populationSets) {
     comparisonData <- data[data$simulationSetName %in% set$simulationSetName, ]
     if (set$isSamePopulation) {
-      logDebug(paste0(
+      logInfo(paste0(
         "Simulation Set '", set$simulationSetName,
         "' was identified with same population as reference '",
         referenceSimulationSetName, "'. ",
         "Ratio comparison analyzed statistics of individual PK ratios."
       ))
       ratioData <- comparisonData
+      # TODO: tlf issue #408
+      # simulationSetName is factor class including multiple levels
+      # After selecting the comparison data, only 1 level is remaining
+      # leading to empty factor levels for calculation of BoxWhisker measure
+      ratioData$simulationSetName <- factor(ratioData$simulationSetName)
       ratioData$Value <- comparisonData$Value / referenceData$Value
       ratioPKMeasure <- getPKParameterMeasure(ratioData, dataMapping)
       ratioPKTable <- rbind.data.frame(ratioPKTable, ratioPKMeasure)
       next
     }
-    logDebug(paste0(
+    logInfo(paste0(
```

Comment:
Previously the comparison between analytical solution and monte carlo simulations was silently saved in log-debug.
Now it will be displayed on the console and saved in log-info